packages/dcos-integration-test/extra/test_meta.py

```diff
@@ -0,0 +1,110 @@
+"""
+Tests for the integration test suite itself.
+"""
+
+import logging
+import os
+import subprocess
+from collections import defaultdict
+from pathlib import Path
+from typing import Set
+
+import yaml
+
+from get_test_group import patterns_from_group
+
+
+log = logging.getLogger(__file__)
+
+
+def _tests_from_pattern(ci_pattern: str) -> Set[str]:
+    """
+    From a CI pattern, get all tests ``pytest`` would collect.
+    """
+    tests = set([])  # type: Set[str]
```

Comment:
Why create an empty list here? `set()` creates an empty set.

Reply:
Good spot! That I think is unnecessary and I will keep this in mind but not change this unless we get blocking comments.